content/quiz_events.md

```diff
@@ -95,6 +95,40 @@ Below are the the scoring calculation rules and the logic for them. This procedu
 **Team Roster**
 : This is an array of quizzer objects, each of which contains a "correct answers" value
 
+### How to Score Individual and Team Points
+**Individual Points**
+
+_**Points Earned**_
+- +20 Points for every correct question & toss-up question
```

Comment:
Do we need to define "question" as "not a toss-up or bonus" ? I think so, because otherwise this could literally be interpreted as the individual gets 20 points for a bonus correct.

Reply:
I think that can be rectified by the added statement commented on your next comment.

All points that occur during a bonus question or during overtime do not contribute to the individual score of a quizzer.